manifests/kots-config.yaml

```diff
@@ -577,6 +577,42 @@ spec:
       description: |-
         Fine-tune your self-hosted environment with advanced configuration.
       items:
+        ## Ai Features
+        ## TODO: replace the links to the docs with the new ones
+        - name: aiFeaturesEnabled
+          title: AI API
+          help_text: |-
+            Enable the AI API. [More info](https://docs.carto.com/carto-self-hosted/guides/enable-the-ai-api).
+          type: bool
+          default: "0"
+
+        ## Litellm
+        - name: enableLitellmEndpoint
+          title: Enable Litellm endpoint
+          help_text: |-
+            Enable the Litellm endpoint.
+          type: bool
+          default: "0"
+        - name: litellmMasterKey
+          title: Litellm master key
+          type: password
+          when: '{{repl (ConfigOptionEquals "aiFeaturesEnabled" "1") }}'
+          required: true
+          default: 'sk-{{repl RandomString 16 "[A-Za-z0-9]" }}'
+        - name: litellmSaltKey
+          title: Litellm salt key
+          type: password
+          when: '{{repl (ConfigOptionEquals "aiFeaturesEnabled" "1") }}'
+          required: true
+          default: 'sk-{{repl RandomString 16 "[A-Za-z0-9]" }}'
+
+        ## Gemini
```

Comment:
IMHO this one should be hidden and we shouldn't allow editing the API key in KOTS. If a customer wants to do it they can do it via extra env var